src/components/FacebookComment.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <main>
+    <div id="fb-root"></div>
+    <div class="fb-comments"></div>
```

Comment:
@alandotic  can you also help add a title so that it would say 'Discuss below' above the comments widget? That would help users understand the context better.